src/shared-components/banner/style.js

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import {
+  COLORS,
+  MEDIA_QUERIES,
+  BOX_SHADOWS,
+  SPACING,
+  TYPOGRAPHY_CONSTANTS,
+} from '../../constants';
+
+const successBannerStyles = css`
+  background-color: ${COLORS.statusGreenBackground};
+  border-color: ${COLORS.statusGreenBorder};
+  color: ${COLORS.statusGreen};
+  fill: ${COLORS.statusGreen};
+`;
+
+const errorBannerStyles = css`
+  background-color: ${COLORS.statusRedBackground};
+  border-color: ${COLORS.statusRedBorder};
+  color: ${COLORS.statusRed};
+  fill: ${COLORS.statusRed};
+`;
+
+const defaultBannerStyles = css`
+  background-color: ${COLORS.statusGreyBackground};
+  border-color: ${COLORS.statusGreyBorder};
+  color: ${COLORS.statusGrey};
+  fill: ${COLORS.statusGrey};
+`;
+
+export const BannerContainer = styled.div`
+  align-items: flex-start;
+  border-width: 1px;
+  border-style: solid;
+  box-shadow: ${BOX_SHADOWS.message};
```

Comment:
does the banner need box shadow? just curious since it's not the exact same as alerts in that it floats above things. Maybe they do?

Reply:
The design shows it as having shadows 
![screen shot 2019-02-14 at 2 49 17 pm](https://user-images.githubusercontent.com/3611/52822839-bea8a980-3067-11e9-85c3-e55f0d0d27e4.png)